examples/acquiring_data/downloading_hmi.py

```diff
@@ -23,23 +23,22 @@
 
 result = Fido.search(a.Time('2014/11/20 00:00:00', '2014/11/20 00:04:00'),
                      a.jsoc.Series("hmi.M_720s"),
-                     a.jsoc.Keys(["T_REC, CROTA2"]),
                      a.jsoc.Notify("jsoc@cadair.com"))
 
 ###############################################################################
 # Now we can see what results we obtained from our search.
-# Notice we have two files.
 print(result)
 
+###############################################################################
+# We can even look at the values of specific keywords from this result.
+jsoc_result = result.get_response(0)
```

Comment:
So we want to remove this, what are our options?

Reply:
I can't use `result['hec']` because the `__get_item__` was defined for `UnifiedResponse` to return an instance of `UnifiedResponse`. Thus I'm using `result.get_response('jsoc')`. I return a list if more than one jsoc records are there, else a `JSOCResponse`.

Reply:
yes. both `jsoc` and `JSOCClient` will work.
It is helpful if there are multiple clients in fido results, because we may not like to count the index to get individual qresponse.